src/ai.ts

```diff
@@ -20,6 +21,15 @@ const LLM_MODELS = [
     name: "gpt-4o-mini",
     createAi: createOpenAI,
   },
+  // Google
+  {
+    name: "gemini-1.5-flash",
+    createAi: createGoogleGenerativeAI,
+  },
+  {
+    name: "gemini-1.5-flash-latest",
+    createAi: createGoogleGenerativeAI,
+  },
```

Comment:
Consider adding version validation or error handling for the Gemini models. Since these are new models, it would be beneficial to:
1. Add version checks to ensure compatibility with the Google AI SDK version being used (`@ai-sdk/google@0.0.55`)
2. Include error handling for cases where the model might not be available
3. Consider adding comments documenting any specific requirements or limitations of these models